samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/TabBarSamplePage.xaml

```diff
@@ -482,6 +482,63 @@
 								 AutomationProperties.AutomationId="MaterialBottomTab3_Count"
 								 Text="{Binding Data.MaterialBottomTab3Count}" />
 					</TextBlock>
+
+
+
+					<utu:TabBar
+					Style="{StaticResource MaterialBottomTabBarStyle}">
```

Comment:
Format xaml I think ;)

Reply:
oh this is just for testing purposes, this tabbar won't stay here. I didn't realize i committed it though. Will be gone once I find the crash